### DISCUSSION TITLE
What if I am having issues cloning because of dependencies ?

### DISCUSSION THREAD

**Discussion by Villex-code** [SheetAble/SheetAble#63] [Q&A] [answered]
I clone and I get a lot of errors , is there something I should look for ?

**Comment by didelik** [comment, marked as answer]
Two of the most common ways to handle collisions in a hashmap are through chaining and open addressing.

Chaining: This is where each bucket of your hashmap points to a linked list of entries that have hashed to the same bucket. When a collision occurs, you simply add the new entry to the end of the list. This can be quite efficient, but if many collisions happen at the same bucket, your lookup time can start to approach O(n), where n is the number of entries in the bucket.

Open Addressing: In open addressing, when a collision occurs, you find another open slot in your table to hold the item that caused the collision. One popular method for open addressing is linear probing, where you simply look to the next slot in the table. If that slot is full, you move to the next one, and so on, until you find an open slot.

**Comment by vallezw** [comment]
@Villex-code do you still have any problems or did the response of didelik solve it?

**Reply by Villex-code** [reply]
Yeah all good , thank you !
